falcon/status_codes.py

```diff
@@ -61,14 +62,18 @@
 HTTP_416 = HTTP_REQUESTED_RANGE_NOT_SATISFIABLE = '416 Range Not Satisfiable'
 HTTP_417 = HTTP_EXPECTATION_FAILED = '417 Expectation Failed'
 HTTP_418 = HTTP_IM_A_TEAPOT = "418 I'm a teapot"
+HTTP_421 = HTTP_MISDIRECTED_REQUEST = "421 Misdirected Request"
 HTTP_422 = HTTP_UNPROCESSABLE_ENTITY = '422 Unprocessable Entity'
 HTTP_423 = HTTP_LOCKED = '423 Locked'
 HTTP_424 = HTTP_FAILED_DEPENDENCY = '424 Failed Dependency'
+HTTP_425 = HTTP_TOO_EARLY = '425 Too Early'
```

Comment:
To my understanding, at the time of writing, `HTTP 425` is still not widely supported by web browsers and HTTP clients. We'd probably need to check the standards track for it, but I'm not opposed for including 425 in Falcon.
Or is it still a bit Too Early for it?

@kgriffs thoughts?